java/cdi/src/main/java/module-info.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
The directory structure you picked is *very* confusing. I expect people seeing a `java` directory will assume all java code is in there, but that's not true here.

I'd suggest `integrationtest/java/modules` instead, but that might require moving around some other integration tests.

Reply:
I've done that and moved the WF tests to the same test directory. I wasn't sure about the TCK tests as that's a strange beast 🧟 😃  so I kept those in the root